src/makeInfinityAdder.js

```diff
@@ -4,9 +4,22 @@
  *
  * @return {function}
  */
+let sum = 0;
```

Comment:
we don't need this variable because we can add our values to variable from parameter in **makeInfinityAdder(x)**